exercises/twelve-days/README.md

```diff
@@ -0,0 +1,119 @@
+Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a
+Partridge in a Pear Tree.
+
+On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming,
+six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two
+Turtle Doves, and a Partridge in a Pear Tree.
+
+On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking,
+seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds,
+three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.
+
+On the ninth day of Christmas my true love gave to me, nine Ladies Dancing,
+eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold
+Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge
+in a Pear Tree.
+
+On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine
+Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six
+Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two
+Turtle Doves, and a Partridge in a Pear Tree.
+
+On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping,
+ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven
+Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three
+French Hens, two Turtle Doves, and a Partridge in a Pear Tree.
+
+On the twelfth day of Christmas my true love gave to me, twelve Drummers
+Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight
+Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings,
+four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a
+Pear Tree.
+```
+
+## Getting Started
+
+For installation and learning resources, refer to the
+[exercism help page](http://exercism.io/languages/haskell).
+
+## Running the tests
+
+To run the test suite, execute the following command:
+
+```bash
+stack test
+```
+
+#### If you get an error message like this...
+
+```
+No .cabal file found in directory
+```
+
+You are probably running an old stack version and need
+to upgrade it.
+
+#### Otherwise, if you get an error message like this...
+
+```
+No compiler found, expected minor version match with...
+Try running "stack setup" to install the correct GHC...
+```
+
+Just do as it says and it will download and install
+the correct compiler version:
+
+```bash
+stack setup
+```
+
+## Running *GHCi*
+
+If you want to play with your solution in GHCi, just run the command:
+
+```bash
+stack ghci
+```
+
+## Feedback, Issues, Pull Requests
+
+The [exercism/haskell](https://github.com/exercism/haskell) repository on
+GitHub is the home for all of the Haskell exercises.
+
+If you have feedback about an exercise, or want to help implementing a new
+one, head over there and create an issue.  We'll do our best to help you!
+
+## Hints
+- Try to capture the structure of the song in your code, where you build up the
```

Comment:
Do you want me to remove this? Keep it?

Reply:
> > HINTS.md
>
> Do you want me to remove this? Keep it?

Sicne it's no longer in HINTS.md, then it needs to be removed from here too, otherwise the README doesn't match what's generated.